docker/Dockerfile

```diff
@@ -0,0 +1,82 @@
+FROM debian:stretch as conpot-builder
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Add non-free packages, needed for snmp-mibs-downloader
+RUN sed -i -e 's/main/main non-free contrib/g' /etc/apt/sources.list
+
+# Install dependencies
+RUN apt-get update -y -qq && apt-get install --no-install-recommends -y -qq \
+        ipmitool \
+        tcpdump \
+        git \
+        python3-pip \
+        python3 \
+        python3-dev \
+        python3-setuptools \
+        build-essential &&\
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Copy the app from the host folder (probably a cloned repo) to the container
+RUN useradd -m conpot
+WORKDIR /opt/
+RUN git clone https://github.com/mushorg/conpot.git
+RUN chown conpot:conpot -R /opt/conpot
+
+# Install Python requirements
+USER conpot
+ENV PATH=$PATH:/home/conpot/.local/bin
+WORKDIR /opt/conpot
+RUN pip3 install --user --no-cache-dir coverage
+RUN pip3 install --user --no-cache-dir -r requirements.txt
+
+# Run test cases
+RUN py.test -v
+
+# Install the Conpot application
+RUN python3.5 setup.py install --user --prefix=
+
+WORKDIR /home/conpot
+RUN tar -zcf conpot.tgz .local
```

Comment:
I think you can just copy over the folder, no need to zip it up

Reply:
yes I can :) you're right

Reply:
fixed